code/game/machinery/camera/camera_assembly.dm

```diff
@@ -23,6 +23,19 @@
 	QDEL_LIST(upgrades)
 	return ..()
 
+/obj/item/camera_assembly/examine(mob/user)
+	. = ..()
+	switch(state)
+		if(ASSEMBLY_UNBUILT)
+			. += "<span class='notice'>The camera's <i>bolts</i> need to be secured in a wall.</span>"
+		if(ASSEMBLY_WRENCHED)
+			. += "<span class='notice'>The camera is <b>bolted</b>, but it needs to be <i>welded</i> into place.</span>"
+		if(ASSEMBLY_WELDED)
+			. += "<span class='notice'>The camera is <b>welded</b> to the wall, it's lacking <i>wires</i>.</span>"
+		if(ASSEMBLY_WIRED)
+			. += "<span class='notice'>The camera is <b>wired</b>, but the maintenence panel needs to be <i>screwed shut</i>.</span>"
+			. += "<span class='notice'>Upgrades can be added to the camera, and removed with a crowbar.</span>"
```

Comment:
Not exactly related, but you probably want to add these examines to the actual camera too, since isn't immediately deconstructed into the assembly